File: led/led_gpio.py
import RPi.GPIO as GPIO
import time

GPIO.setmode(GPIO.BCM)

LED = (13,19,26)

GPIO.setup(LED, GPIO.OUT)

try:
    i = 0
    while True:
        print('LED %s on' % i)
        GPIO.output(LED[i], GPIO.HIGH)
        time.sleep(1.5)

        print('LED %s off' % i)
        GPIO.output(LED[i], GPIO.LOW)
        time.sleep(1.5)
        i = (i + 1) % 3

except KeyboardInterrupt:
    pass

print('pin clean')
GPIO.cleanup()

File: led/led_gpio.c
#include <wiringPi.h>
#include <stdio.h>
#include <unistd.h>
#include <stdlib.h>

// #define BCM

int main(void){
    #ifndef BCM
    // printf("In wiringPi code.");
    char LED[3] = {23,24,25};
	if(wiringPiSetup() < 0){
	    perror("Start GPIO Failed.");
	    exit(1);
	}
	for(int i = 0;i<3;i++)
        pinMode(LED[i],OUTPUT);
	while(1){
	    for(int i = 0;i<3;i++){
            digitalWrite(LED[i],1);
            sleep(1);
            digitalWrite(LED[i],0);
        }
	}
    #endif
    #ifdef BCM
    // printf("In BCM code.");
    char LED[3] = {13,19,26};
    if(wiringPiSetupGpio() < 0){
        perror("Start GPIO Failed.");
	    exit(1);
    }
    for(int i = 0;i<3;i++)
        pinMode(LED[i],OUTPUT);
	while(1){
	    for(int i = 0;i<3;i++){
            digitalWrite(LED[i],1);
            sleep(1);
            digitalWrite(LED[i],0);
        }
	}
    #endif
}